theforeman.org/pipelines/lib/copr.groovy

```diff
@@ -40,15 +40,11 @@ def copr_repos(package_name) {
 }
 
 def convert_to_dist(chroot) {
-    if(chroot == 'rhel-9-x86_64') {
-        return 'el9'
-    } else if(chroot == 'rhel-8-x86_64') {
-        return 'el8'
-    } else if(chroot == 'rhel-7-x86_64') {
-        return 'el7'
-    } else if(chroot == 'opensuse-leap-15.5-x86_64') {
-       return 'leap155'
-    } else {
-        return null
-    }
+    def family_map = ['rhel': 'el', 'centos-stream': 'el', 'opensuse-leap': 'leap']
+
+    chroot_parts = chroot.split('-')
+    version = chroot_parts[-2].replace('.', '')
+    os = chroot_parts[0..-3].join('-')
```

Comment:
My brain prefers this the other way around so you first take the beginning of the string and then the end.
```suggestion
    os = chroot_parts[0..-3].join('-')
    version = chroot_parts[-2].replace('.', '')
```

Reply:
your brain is weird :P

Reply:
Tell me something I don't know ;)